include/irr/asset/IAssetLoader.h

```diff
@@ -7,9 +7,47 @@
 namespace irr { namespace asset
 {
 
+//! A class that defines rules during Asset-loading process
+/**
+	Every Asset must be loaded by a particular class derived from IAssetLoader.
```

Comment:
Not true, the particular assets can be instantiated and filled out by hand. Loaders just automate this process and can create assets from "files" (or any other resource interfacing as a file).

Reply:
status?

Reply:
Solved